README.rst

```diff
@@ -0,0 +1,47 @@
+python-ethtool
+==============
+
+*Python bindings for the ethtool kernel interface*
+
+**This is the new upstream for python-ethtool maintained by Fedora's
+Python SIG.**
+
+About
+-----
```

Comment:
I'd suggest an *About* section is overkill, just merge it with the perex above. I'd personally move the next paragraph above the `**This is the new upstream...` line.